active/InformationBar/InformationBar.md

```diff
@@ -0,0 +1,512 @@
+  - Additionally, if a status change to the app is so severe that it needs to block all further ability for the user to interact with the app, use a ContentDialog.
+- For scenarios where a notification is a transient teaching moment, a [TeachingTip](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/dialogs-and-flyouts/teaching-tip) is a better option.
+
+For more info about choosing the right notification control, see the [Dialogs and Flyouts](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/dialogs-and-flyouts/) article.
+
+# Examples
+
+An info bar can have several configurations, here are some notable ones.
+
+When an InfoBar is conveying information of a common severity, like displaying an error or warning, it can be set to have one of many severity levels to use consistent [Fluent](https://docs.microsoft.com/en-us/windows/apps/fluent-design-system) styling for it's identifiers.
+![A mockup of an InfoBar in a Warning state with a close button and a message](images/Warning_NoButton.png)
+
+If a call to action is needed, an info bar can have customizable action or hyperlink buttons.
+![A mockup of an InfoBar](images/Default_MessageHyperlink.png)
+
+The info bar can also be customized with its optional properties and with XAML Content to include extra buttons and other UI elements.
+![A mockup of an InfoBar with custom status color and content](images/Custom_IconColor.png)
+
+## Create an InfoBar
+
+The XAML below describes an inline InfoBar with the default styling for an error  notification. An info bar can be created anywhere in the element tree or code behind. In this example the InfoBar expands to fill the width of the StackPanel layout control it resides in.
+
+By default, the InfoBar will not be visible. Set the IsOpen property to true in the XAML or code behind to display the information.
+
+XAML
+```xml
+<Grid>
+    <!--Other UI content -->
+    <StackPanel x:Name="ContentArea">
+        <controls:InfoBar x:Name="UnsuccessfulSaveNotification"
+            Severity="Warning"
+            Title="Error while saving"
+            Message="Your document was unable to be saved.">
+        </controls:InfoBar>
+    </StackPanel>
+    <!--Other UI content -->
+<Grid>
+```
+C#
+```C#
+public MainPage()
+{
+    this.InitializeComponent();
+
+    if(!SaveSuccessful())
+    {
+        UnsuccessfulSaveNotification.IsOpen = true;
+    }
+}
+```
+
+Here is the visual representation of the info bar in the page.
+
+![A mockup of an InfoBar in a Warning state with a close button and a message](images/Warning_NoButton.png)
+
+## Using pre-defined severity levels
+The type of the info bar can be set via the Severity property to automatically set a consistent status color, icon, and assistive technology settings dependent on the criticality of the notification.
+
+Mockups in various Severities to be added.
```

Comment:
We don't usually work with mockups in the documentation. These would be better as examples (including the markup that creates them).

Reply:
When you mean mockups do you mean the File Explorer sample app screenshots? I can create a simpler sample app to grab screenshots to highlight the functionality if that's valuable at this stage. 
For the content wrapping example, showing just the InfoBar on its own is not as valuable as showing where it is on the page and its relation to other content. For the markup snippet, is it beneficial to include the accompanying layout controls?